src/Domain/Bootstrap.php

```diff
@@ -183,4 +187,44 @@ protected function define_feature_flag() {
 		}
 		define( 'WOOCOMMERCE_BLOCKS_PHASE', $flag );
 	}
+
+	/**
+	 * This is a temporary method that is used for setting up payment method
+	 * integrations with Cart and Checkout blocks. This logic should get moved
+	 * to the payment gateway extensions.
+	 */
+	protected function load_payment_method_integrations() {
+		// stripe registration.
+		$this->container->register(
+			Stripe::class,
+			function( Container $container ) {
+				$asset_api = $container->get( AssetApi::class );
+				return new Stripe( 'stripe', $asset_api );
+			}
+		);
+		// cheque registration.
+		$this->container->register(
+			Cheque::class,
+			function( Container $container ) {
+				$asset_api = $container->get( AssetApi::class );
+				return new Cheque( 'cheque', $asset_api );
+			}
+		);
+		add_action(
+			'wc_blocks_library_payment_method_type_registration',
+			function( $payment_method_registry ) {
+				// This is temporarily registering Stripe until it's moved to the
+				// extension.
+				if ( class_exists( 'WC_Stripe' ) && ! $payment_method_registry->is_registered( 'stripe' ) ) {
+					$payment_method_registry->register(
+						$this->container->get( Stripe::class )
+					);
+				}
+				$payment_method_registry->register(
+					$this->container->get( Cheque::class )
+				);
+			}
+		);
```

Comment:
For demonstration purposes, I'm hooking into the new action, however for built-ins we can probably just pull the `$payment_method_registry` directly from the container and register payment built-in payment method types that way.